cert-manager/config/manager/manager.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: cert-manager
```

Comment:
same reason everything is cert-manager to distinguish from our controller-manager